docs/api-and-integration/octopus.server.exe-command-line/index.md

```diff
@@ -6,32 +6,34 @@ position: 90
 
 **Octopus.Server.exe** is the executable that runs the Octopus Server instance. It includes many helpful commands that allow you to manage the instance, including; authentication, configuration, diagnostics and running the service.
 
-## Commands {#Octopus.Server.exeCommandLine-Commands}
+## Commands {#octopus.server.exeCommandLine-Commands}
 
-`Octopus.Server.exe` supports the following commands:
+`octopus.server.exe` supports the following commands:
 
 - **`admin`**:  Reset admin user passwords, re-enable them, and ensure they are in the admin group.
-- **`builtin-worker`**:  Configure the built-in worker used to run deployment actions and scripts on the Octopus Server.  Available in **Octopus 2018.1.0** and later.
+- **`builtin-worker`**:  Configure the built-in worker used to run deployment actions and scripts on the Octopus Server..
 - **`checkservices`**:  Checks the Octopus instances are running.
 - **`configure`**:  Configure this Octopus instance.
 - **`create-instance`**:  Registers a new instance of the Octopus service.
 - **`database`**:  Create or drop the Octopus database.
 - **`delete-instance`**:  Deletes an instance of the Octopus service.
-- **`export-certificate`**:  Exports the certificate that Octopus Server can use to authenticate itself with its Tentacles.
-- **`external-worker`**:  Configure the external worker used to run deployment actions and scripts on the Octopus Server. Available in **Octopus 2018.2.0** and later.
+- **`export-certificate`**:  Exports the certificate that Octopus server can use to authenticate itself with its Tentacles.
 - **`help`**:  Prints this help text.
-- **`import-certificate`**:  Replace the certificate that Octopus Server uses to authenticate itself with its Tentacles.
+- **`import-certificate`**:  Replace the certificate that Octopus server uses to authenticate itself with its Tentacles.
```

Comment:
Need to fix the capitalisation

Reply:
fixed by https://github.com/OctopusDeploy/OctopusDeploy/commit/e515237db97c54b5a97b1b22084d8f2584069d56